include/mbedtls/build_info.h

```diff
@@ -36,6 +59,12 @@
 #include MBEDTLS_CONFIG_FILE
 #endif
 
+#if defined(MBEDTLS_CONFIG_VERSION) && ( \
+    MBEDTLS_CONFIG_VERSION < 0x03000000 || \
+    MBEDLTS_CONFIG_VERSION > MBEDTLS_VERSION_NUMBER )
```

Comment:
It's a bit awkward that until the release, people won't be able to try out defining `MBEDLTS_CONFIG_VERSION`, since no version is currently acceptable. But that's only going to last a few days, so I guess it doesn't matter.